resource/desktop/com.github.eneshecan.WhatsAppForLinux.desktop.in

```diff
@@ -13,6 +13,7 @@ Name[nl]=WhatsApp voor Linux
 Name[pt-br]=WhatsApp para Linux
 Name[ru]=WhatsApp для Linux
 Name[tr]=Linux için WhatsApp
+Name[si]=ලිනක්ස් සඳහා WhatsApp
```

Comment:
Could you please move this line one level up so that they are alphabetically sorted?